dist/gcc-compatible/EverCrypt_Hash.c

```diff
@@ -402,7 +401,7 @@ void EverCrypt_Hash_init(EverCrypt_Hash_state_s *s)
     r1[1U] = iv5;
     r1[2U] = iv6;
     r1[3U] = iv7;
-    FStar_UInt128_uint128 uu____1 = FStar_UInt128_uint64_to_uint128((uint64_t)0U);
+    (void)FStar_UInt128_uint64_to_uint128((uint64_t)0U);
```

Comment:
just weird! I think this patch makes things worse, but for this specific instance, I've patched krml to "know" that this function is pure, so we can easily remove this quirk

Reply:
(in a future iteration)